.jenkins/scripts/global/init-docker.sh

```diff
@@ -0,0 +1,15 @@
+while getopts o: flag
+do
+  case "${flag}" in
+    u) BUILD_USER=${OPTARG};;
+  esac
+done
+
+if [ -z $BUILD_USER ]; then
+echo "BUILD_USER required. Run script with '-u <BUILD_USER>' flag"; exit 1; fi
+
+sudo usermod -aG docker \${BUILD_USER}
+sudo chmod 666 /var/run/docker.sock
```

Comment:
Please do not do this as it is a big security vulnerability

Reply:
It's a copy and paste from an existing pipeline, not sure if it's needed for the test to work